src/rules/noTripleEqualsNullRule.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+import * as Lint from "../lint";
+
+const OPTION_NO_NULL_KEYWORD = "no-null-keyword";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static EQ_FAILURE_STRING = "Did you mean == null instead?";
+    public static NEQ_FAILURE_STRING = "Did you mean != null instead?";
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        const noTripleEqualsNullWalker = new NoTripleEqualsNullWalker(sourceFile, this.getOptions());
+        return this.applyWithWalker(noTripleEqualsNullWalker);
+    }
+}
+
+class NoTripleEqualsNullWalker extends Lint.RuleWalker {
+    public visitBinaryExpression(node: ts.BinaryExpression) {
+        if (this.isExpressionAllowed(node)) {
+            const position = node.getChildAt(1).getStart();
+            const expressionWidth = node.right.getFullWidth() + 3;
```

Comment:
Probable better to do something like: `const expressionWidth = node.right.getEnd() - position;` in case there's extra whitespace lying around.